src/cc/libbpf.c

```diff
@@ -513,13 +513,11 @@ void * bpf_attach_uprobe(int progfd, enum bpf_probe_attach_type attach_type, con
 {
   int kfd;
   char buf[PATH_MAX];
-  char new_name[256];
+  char event_alias[PATH_MAX];
   struct perf_reader *reader = NULL;
   static char *event_type = "uprobe";
-  int ns_fd = -1;
-  int n;
+  int res, ns_fd = -1;
 
-  snprintf(new_name, sizeof(new_name), "%s_bcc_%d", ev_name, getpid());
   reader = perf_reader_new(cb, NULL, NULL, cb_cookie, probe_perf_reader_page_cnt);
   if (!reader)
     goto error;
```

Comment:
If returning error here, kfd will be uninitialized. So you may want to initialize kfd = -1.